docs/ManualDeploymentGuideScalarDBServerOnEKS.md

```diff
@@ -0,0 +1,200 @@
+* Configure kubectl to connect to your Kubernetes cluster using the `aws eks update-kubeconfig` command.
+
+   ```console
+   aws eks --region <region-code> update-kubeconfig --name <cluster_name>
+   ```
+
+## Step 4. Install Scalar DB server
+
+After creating a Kubernetes cluster, the next step is to deploy the Scalar DB server into the EKS cluster.
+This section shows how to install the Scalar DB server to the EKS cluster with [Helm charts](https://github.com/scalar-labs/helm-charts).
+
+### Prerequisites
+
+Install the Helm on your bastion server to deploy helm charts:
+
+* [Helm](https://helm.sh/docs/intro/install/): Helm is a command-line tool to manage releases in the EKS cluster. In this tutorial, it is used to deploy the Scalar DB server helm chart to the EKS cluster. Helm version 3.5 or the latest is required.
+
+### Steps
+
+The following steps show how to install the Scalar DB server on EKS:
+
+1. Download the following Scalar DB server configuration files from the [scalar-kubernetes repository](../conf).
+   Note that they may be versioned in the future, so you might want to change the branch to use a proper version.
+    * scalardb-custom-values.yaml
+
+2. Update the database configuration in scalardb-custom-values.yaml  sections as specified in [configure Scalar DB guide](./ConfigureScalarDB.md).
+3. Run the Helm commands on the bastion server to install the Scalar DB server on EKS
+
+   ```console
+    # Add Helm charts
+      helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
+    
+    # List the Scalar charts.
+      helm search repo scalar-labs
+    
+    # Install Scalar DB with a release name `my-release-scalardb`
+      helm upgrade --version <chart version> --install my-release-scalardb scalar-labs/scalardb --namespace default -f scalardb-custom-values.yaml
+    ```
+
+Note:
+
+* The same commands can be used to upgrade the pods.
+* Release name `my-release-scalardb` can be changed at your convenience.
+* The `chart version` can be obtained from `helm search repo scalar-labs` output
+* `helm ls -a` command can list currently installed releases.
+
+## Step 5. Monitor the Cluster
+
+It is critical to actively monitor the overall health and performance of the cluster running in production.
+You can use Container Insights to collect performance metrics and Fluent Bit to collect logs of the EKS cluster.
+This section shows how to configure monitoring and logging for your EKS cluster.
+
+### Recommendations
+
+* You should enable monitoring for EKS in production.
+* You should configure the cloudwatch agent in the EKS cluster for collecting metrics from pods.
+* You should configure Fluent Bit in the EKS cluster for collecting logs from pods.
+    * You should create a CloudWatch Logs policy with `CreateLogGroup`, `CreateLogStream`, and `PutLogEvents` permissions and attach the policy to the Node instance role.
```

Comment:
I think this is a requirement when we use the CloudWatch.
Also, the EKS official document explains the prerequisites (includes IAM role configuration).
https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-prerequisites.html

So, I think it is better to move this sentence to the **Requirements** section and refer to this official document.

Reply:
Thank you for the suggestion.
I have created a **Requirements** section and moved this sentence with reference to the official documentation.